### PR TITLE
Style proportions

### PR DESCRIPTION
**Pull request by hippogriffic**
image

https://incuna-pm.atlassian.net/browse/OP-247
@incuna/frontend please review

### CODE REVIEW

File: web/templates/twig-source/incuna-surveys/form/base/proportion-field.html [hunk 1]
```diff
@@ -1,16 +1,34 @@
...
+        Total: <span class="total-figure" ng-bind="total"></span>
+    </span>
+{% endblock %}
+{% block fields %}
+    <div class="fields-wrapper">
+        <div class="proportion-field" ng-repeat="field in fields">
+            {% block field %}
+                <div drf-form-field="field" class="proportion-field-inner {% block field-class %}{% endblock %}">
+                    {% block input %}
+                        <input
```
Comment (maxpeterson) [inline web/templates/twig-source/incuna-surveys/form/base/proportion-field.html]: THis needs to use `<span integer-field model="model[$index]" id="field.id" form="form"></span>` to include the error messages.

Comment (maxpeterson) [inline web/templates/twig-source/incuna-surveys/form/base/proportion-field.html]: If you need the `proportion-input` class then can it be on the span `<span integer-field class="proportion-input" model="model[$index]" id="field.id" form="form"></span>`

Comment (maxpeterson) [inline web/templates/twig-source/incuna-surveys/form/base/proportion-field.html]: @hippogriffic I attempted to add this but gave up on the styling side - it almost worked but probably best if you fixed it

**Comment by maxpeterson** [comment]
@hippogriffic you need to run `grunt watch` or `grunt build` to generate the templates for the library.

**Comment by hippogriffic** [comment]
image

hooked up with form errors

**Pull request merged**